src/tiled-map-resource.ts

```diff
@@ -42,6 +42,18 @@ export enum TiledMapFormat {
    JSON = 'JSON'
 }
 
+export interface TiledMapOptions {
+   /**
+    * Optionally override the map format, by default files ending in .tmx are treated as TMX format, otherise treated as JSON format
+    */
+   mapFormatOverride?: TiledMapFormat;
+
+   /**
+    * Optionally Specify the first layer z-index value, each layer will increment up 1 unless overridden (default -1)
```

Comment:
I'm not sure about the name. Is this number more accurately the starting z-index for auto incrementing values? If the first layer overrides it's z-index, this is confusingly named.

```suggestion
    * Override the starting auto-incrementing z-index value (default: `-1`). Each layer will increment this number by 1 unless the layer specifies it's own custom `zindex` property.
```